src/site/topic7.rst

```diff
@@ -2,6 +2,18 @@
 Topic #7 --- Links
 ******************
 
+.. warning::
+
+    We are not yet defining a new ADT. We're just prepping to start implementing other ADTs with something other than an array.
+
+
+* Arrays have a fixed size
+    * But we can expand capacity
+
+* When using an array, we may have to shift things around when adding and removing
+* Traditionally speaking, arrays are in contiguous memory addresses
+    * JVM asterisk
```

Comment:
Just some sources:

[java specification: arrays](https://docs.oracle.com/javase/specs/jls/se11/html/jls-10.html)
> In the Java programming language, arrays are objects (§4.3.1), are dynamically created, and may be assigned to variables of type Object (§4.3.2). All methods of class Object may be invoked on an array.

[jvm specification](https://docs.oracle.com/javase/specs/jvms/se11/html/jvms-2.html#jvms-2.5.3)
> The Java Virtual Machine has a heap that is shared among all Java Virtual Machine threads. The heap is the run-time data area from which memory for all class instances and arrays is allocated
> ...
> The memory for the heap does not need to be contiguous.

TLDR: Objects go into the heap, arrays are objects so are in the heap. The heap isn't necessarily contiguous.

Reply:
Adding the TLDR to point on JVM asterisk.